### PR TITLE
feat: support usage of consistency parameter

### LINKED ISSUE #379: Add support for --consistency parameter in queries
OpenFGA now provides a way to specify a `consistency` parameter for queries.

We should add it to the CLI, e,g,  `fga query ... --consistency "higher_consistency"`

### PR DESCRIPTION
**Pull request by ewanharris**
## Description

Adds support for the `--consistency` parameter to the query commands to allow usage of that feature.

For more details see the [relevant openfga/openfga PR](https://github.com/openfga/openfga/pull/1764).

## References

Closes #379

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

**Review by rhamzeh — APPROVED** [review]
Thanks! Do you think we can add this to the README (in a follow-up PR?)

**Comment by ewanharris** [comment]
@rhamzeh, is there anything in addition to the [existing changes](https://github.com/openfga/cli/pull/381/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) that I forgot?

**Comment by rhamzeh** [comment]
@ewanharris sorry I meant if we can add that to the examples in the command - not the readme.

e.g.
- https://github.com/openfga/cli/blob/main/cmd/query/check.go#L59
- https://github.com/openfga/cli/blob/main/cmd/query/expand.go#L49
- https://github.com/openfga/cli/blob/main/cmd/query/list-objects.go#L61
- https://github.com/openfga/cli/blob/main/cmd/query/list-relations.go#L124
- https://github.com/openfga/cli/blob/main/cmd/query/list-users.go#L91
- https://github.com/openfga/cli/blob/main/cmd/tuple/read.go#L127

**Comment by ewanharris** [comment]
Putting into draft to avoid merging as currently missing the `fga tuple read` command support

File: cmd/store/export.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func buildStoreData(config fga.ClientConfig, fgaClient client.SdkClient, maxTupl
 	// get the tuples
 	maxPages := int(math.Ceil(float64(maxTupleCount) / float64(tuple.DefaultReadPageSize)))
 
-	rawTuples, err := tuple.Read(fgaClient, &client.ClientReadRequest{}, maxPages)
+	rawTuples, err := tuple.Read(fgaClient, &client.ClientReadRequest{}, maxPages, nil)
```
[the remark this answers is not in the document]
Comment (rhamzeh) [reply cmd/store/export.go:71]: Hmm. I'm like you. I think maybe export should always be high consistency 🤔

Let's discuss it

[the remark this answers is not in the document]
Comment (ewanharris) [reply cmd/store/export.go]: Given we've an initial agreement, I pushed eb1636657a122d33f667c5418b0e84574b0d4367. Can always revert if we decide not to

[the remark this answers is not in the document]
Comment (aaguiarz) [reply cmd/store/export.go]: Not sure about export, at the end of the day it's always a snapshot in time, and the data might be different 10 seconds later. Higher consistency reads will be way more expensive for little gain.

[the remark this answers is not in the document]
Comment (jimmyjames) [reply cmd/store/export.go:71]: Reverted https://github.com/openfga/cli/commit/eb1636657a122d33f667c5418b0e84574b0d4367 to not export with high consistency by default as discussed.

**Review by rhamzeh — APPROVED** [review]

**Pull request merged**